build/shade/_k-standard-goals.shade

```diff
@@ -157,14 +155,11 @@ default NUGET_FEED = 'https://api.nuget.org/v3/index.json'
   copy sourceDir='${Path.GetDirectoryName(project)}' include='bin/**/' outputDir='${Path.Combine(BUILD_DIR, Path.GetFileNameWithoutExtension(project))}' overwrite='${true}' each='var project in nativeProjects'
 
 #nuget-verify target='package' if='File.Exists(PACKAGELIST_JSON_FILENAME) && ShouldVerifyNupkgs' description='Verify if all the packages are generated properly'
-  var commandsDirectory = '${Path.Combine(BASE_DIR, "commands")}'
-  exec program='.build/NuGet.exe' commandline='install -ExcludeVersion -pre NuGetPackageVerifier -Source ${DNX_TOOLS_FEED} -out ${commandsDirectory}'
-  exec program='${Path.Combine(commandsDirectory, "NuGetPackageVerifier", "NuGetPackageVerifier.exe")}' commandline='"${BUILD_DIR}" "${Path.Combine(BASE_DIR, PACKAGELIST_JSON_FILENAME)}"'
   @{
-    if (Directory.Exists(commandsDirectory))
-    {
-      Directory.Delete(commandsDirectory, recursive: true);
-    }
+    var verifierDirectory = Directory.EnumerateDirectories(Path.Combine(KoreBuildFolderPath, "NuGetPackageVerifier")).First();
+    Exec(
+        Path.Combine(verifierDirectory, "NuGetPackageVerifier.exe"),
+        string.Format("\"{0} \" \"{1} \"", BUILD_DIR, Path.Combine(BASE_DIR, PACKAGELIST_JSON_FILENAME)));
```

Comment:
Realize this PR has been merged. But are the spaces at the end of each quoted argument intentional?